src/react-apps/ux-editor/src/sagas/formDesigner/formDesignerSagas.ts

```diff
@@ -11,30 +11,66 @@ const uuid = require('uuid/v4');
 const selectFormDesigner = (state: IAppState): IFormDesignerState => state.formDesigner;
 const selectFormFiller = (state: IAppState): IFormFillerState => state.formFiller;
 
+function* addActiveFormContainerSaga({
+  containerId,
+  callback,
+}: FormDesignerActions.IAddActiveFormContainerAction): SagaIterator {
+
+  try {
+    const formDesignerState: IFormDesignerState = yield select(selectFormDesigner);
+
+    if (containerId === formDesignerState.layout.activeContainer) {
```

Comment:
You meant like this?
`function* addActiveFormContainerSaga({ containerId }: FormDesignerActions.IAddActiveFormContainerAction): SagaIterator {...`

Reply:
No just the yield call part, so it can be written like:
yield call(FormDesignerActionDispatchers.addActiveFormContainerFulfilled, containerId === formDesignerState.layout.activeContainer ? '' : containerId)